build_windows.py

```diff
@@ -17,139 +17,140 @@
 
 # List of dlls that cx_freeze can't detect automatically
 missing_dll = [
```

Comment:
Excellent thanks and approved.

let's add a pipeline test that rejects merges if the code isn't blackened.